apps/checker/app/services/collins/CollinsEnglish.java

```diff
@@ -16,6 +16,11 @@
 import java.util.ResourceBundle;
 
 public class CollinsEnglish extends BritishEnglish {
+
+    public CollinsEnglish() {
+        super(false);
```

Comment:
The argument to the `super()` isn't really used.  The new version of LanguageTool creates another constructor for subclasses to instantiate directly, and the argument is just intended to make it different from the default constructor.